easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'PostgreSQL'
+version = '9.5.2'
+
+homepage = 'http://www.mysql.com/'
+description = """PostgreSQL is a powerful, open source object-relational database system.
+ It is fully ACID compliant, has full support for foreign keys,
+ joins, views, triggers, and stored procedures (in multiple languages).
+ It includes most SQL:2008 data types, including INTEGER,
+ NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
+ It also supports storage of binary large objects, including pictures,
+ sounds, or video. It has native programming interfaces for C/C++, Java,
+ .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation."""
+
+source_urls = ['http://ftp.postgresql.org/pub/source/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+java = 'Java'
+javaver = '1.8.0_77'
+
+dependencies = [
+    (java, javaver, '', True),
+    ('libreadline', '6.3'),
+    ('zlib', '1.2.8'),
+    ('Python', '2.7.11'),
```

Comment:
for Python, we do it consistently, I feel it helps in making it clear what other (Python) tools/libraries can be combined with this module